CHANGELOG.md

```diff
@@ -0,0 +1,62 @@
+# API changes in Preview 3:
+
+🆕 New `ObservableValidator` class, which supports the [`INotifyDataErrorInfo`](https://docs.microsoft.com/dotnet/api/system.componentmodel.inotifydataerrorinfo) interface.
+
+🆕 Added new constructors with cancellation support to the async commands, and added new cancellation-related properties to the async command interfaces.
+
+🆕 Added a new `WeakReferenceMessenger` type. This type is less performant than the other messenger, and uses more memory, and in return only uses weak references to track recipients. This type essentially mirrors the behavior of the `Messenger` type from `MvvmLight`, making the transition easier for developers migrating from that library.
+
+🆕 Introduced a new custom delegate to represent message handlers, which also receives the current recipient as additional input parameter. No API changes are required for users registering messages through the `IRecipient<TMessage>` interface, whereas users manually registering will need to modify their code as follows:
```

Comment:
Do we want to mark where there are required code changes, like here and the SetPropertyAndNotifyOnCompletion below?

Reply:
I do indicate the changes in the code snippet right after this point, do you mean something else? 🤔

Reply:
Reorganized that section in e954634ed26aef9f4cd1e8815812325f7da7af4f, let me know what you think! 😊